tests/unit/test_data_cleaning_workflow.py

```
@@ -0,0 +1,272 @@
+        'group': [1, 2, 1, 2, 1, 2],
+        'outcome': ["Yes", "No", "Yes", "No", "Yes", "No"]
+    })
+    
+    print(f"\nOriginal DataFrame:\n{original_data}")
+    print(f"\nOriginal dtypes:\n{original_data.dtypes}")
+    
+    # Store original values for comparison
+    original_age_0 = original_data.loc[0, 'age']
+    original_weight_4 = original_data.loc[4, 'weight']
+    
+    print(f"\nStored original values:")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)